odoo-shippable/scripts/build-image.sh

```diff
@@ -420,7 +420,7 @@ cp -r ${HOME}/.tmux.conf /home/odoo
 chown -R odoo:odoo /home/odoo/.tmux.conf
 # Install all plugin for all user
 ${HOME}/.tmux/plugins/tpm/scripts/install_plugins.sh
-/home/odoo/.tmux/plugins/tpm/scripts/install_plugins.sh
+su odoo /home/odoo/.tmux/plugins/tpm/scripts/install_plugins.sh
```

Comment:
What about install the plugin for both users (root and odoo)?

Reply:
@moylop260 
The script build-image.sh ran with the user root only we need to run with the user odoo.
In fact the tmux-resurrect plugin actually is correctly installed on root

In the line 422 of the file build-image.sh the command install_plugins.sh run with the root user